hibernate-core/src/main/java/org/hibernate/tool/schema/extract/spi/ExtractionContext.java

```diff
@@ -31,7 +33,15 @@
 public interface ExtractionContext {
 	ServiceRegistry getServiceRegistry();
 	JdbcEnvironment getJdbcEnvironment();
-	SqlStringGenerationContext getSqlStringGenerationContext();
+	default SqlStringGenerationContext getSqlStringGenerationContext() {
```

Comment:
Same as the default in `SessionFactoryImplementor`; I wouldn't add it here unless there's a very good, concrete reason.